packages/@misk/simpleredux/src/dispatch.ts

```diff
@@ -10,10 +10,12 @@ import {
 
 /**
  * Configurable options for each Dispatch function
- * @param mergeSaga: function to run after the saga to yield more actions
+ * @param failureSaga: optional saga to run after the handled failure action to yield more actions
+ * @param mergeSaga: optional saga to run after the handled merge action to yield more actions
```

Comment:
Would `mergeSaga` run before or after the request gets sent out?

Reply:
`mergeSaga` and `failureSaga` always run after the initial action is handled. So if you dispatch a `simpleHttpPost`, success or failure would lead to running `mergeSaga` and `failureSaga` respectively if they are passed in the `simpleHttpPost` options.